blaze/expr/collections.py

```diff
@@ -97,8 +102,17 @@ class Distinct(Expr):
     >>> from blaze.compute.python import compute
     >>> sorted(compute(e, data))
     [('Alice', 100, 1), ('Bob', 200, 2)]
+
+    Using a subset
```

Comment:
You have to put a space after these comments otherwise sphinx won't render them correctly.